runtime/cere-dev/Cargo.toml

```diff
@@ -101,7 +101,7 @@ pallet-erc20 = { version = "2.0.0", default-features = false, path = "../../pall
 pallet-ddc-metrics-offchain-worker = { version = "2.0.0", default-features = false, path = "../../pallets/ddc-metrics-offchain-worker" }
 
 [build-dependencies]
-substrate-wasm-builder = { version = "5.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22" }
+substrate-wasm-builder = { version = "5.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23" }
```

Comment:
Update [try-runtime] section with `pallet-bags-list`

Reply:
We already had pallet-bags-list/try-runtime in try-runtime

Reply:
Please, send to me a link to the line. I can not find it.

Reply:
Sorry. I was checking the wrong place. Added.